src/xo-app/logs/index.js

```diff
@@ -70,9 +73,36 @@ class JobReturn extends Component {
   }
 }
 
+const getJobCallStateInfos = (end, error) => {
+  const {
+    icon,
+    tooltip
+  } = error !== undefined
+    ? {icon: 'halted', tooltip: 'failedJobCall'}
+    : end !== undefined
+      ? {icon: 'running', tooltip: 'successfulJobCall'}
+      : {icon: 'busy', tooltip: 'jobCallInProgess'}
+
+  return <Tooltip content={_(tooltip)}>
+    <Icon icon={icon} />
+  </Tooltip>
+}
+
+const getJobTransferredDataInfos = (start, end, size) => <div>
+  <span><strong>{_('jobTransferredDataSize')}</strong> {formatSize(size)}</span>
+  <br />
+  <span><strong>{_('jobTransferredDataSpeed')}</strong> {formatSpeed(size, end - start)}</span>
```

Comment:
Both the label and the value should be translatable because you have no guarantee that the order will stay the same in every languages.

We'll keep this as is for now but we'll have to fix this later.